src/main.rs

```diff
@@ -280,7 +280,7 @@ fn concat_vendor_css(files: Vec<&str>) {
 fn main() {
     compile_sass("app");
     compile_sass("fonts");
-    concat_vendor_css(vec!["skeleton", "tachyons"]);
+    concat_vendor_css(vec!["tachyons"]);
```

Comment:
Please remove this change from the patchset, as other pages are not fixed yet :).

Reply:
_faceplam_

I made a specific mental note about this when I started working on the file and then pushed it all the same lol